gp-includes/strings.php

```diff
@@ -217,3 +217,89 @@ function gp_levenshtein( $str1, $str2, $length1, $length2 ) {
 
 	return $prevRow[$length2];
 }
+
+/**
+ * Sanitizes a project name, replacing whitespace and a few other characters with dashes.
+ *
+ * Limits the output to alphanumeric characters, underscore (_), periods (.) and dash (-).
+ * Whitespace becomes a dash.
+ *
+ * @since 2.1.0
+ *
+ * @param string $project_name The project name to be sanitized.
+ *
+ * @return string The sanitized title.
+ */
+function gp_sanitize_project_name( $project_name ) {
+	$project_name = strip_tags( $project_name );
```

Comment:
The first line should be `$project_name = remove_accents( $project_name );`, see `sanitize_title()`.